backend/src/test/java/com/woowacourse/levellog/acceptance/AcceptanceTest.java

```diff
@@ -101,21 +106,10 @@ private void setRestDocsSpec(final RestDocumentationContextProvider contextProvi
                 ).build();
     }
 
-    @Deprecated
-    protected RestAssuredResponse requestCreateTeam(final String title, final String token,
-                                                    final Long... participantIds) {
-        final ParticipantIdsDto participantIdsDto = new ParticipantIdsDto(List.of(participantIds));
-        final TeamWriteDto request = new TeamWriteDto(title, title + "place", 1, TEAM_START_TIME,
-                participantIdsDto);
-
-        return post("/api/teams", token, request);
-    }
-
-    @Deprecated
     protected RestAssuredResponse login(final String nickname) {
         try {
-            final GithubProfileDto response = new GithubProfileDto(String.valueOf(
-                    ((int) System.nanoTime())), nickname, nickname + ".com");
+            final GithubProfileDto response = new GithubProfileDto(
+                    String.valueOf(((int) System.nanoTime())), nickname, nickname + ".com");
```

Comment:
```suggestion
            final GithubProfileDto response = new GithubProfileDto(
                    String.valueOf((int) System.nanoTime()), nickname, nickname + ".com");
```
괄호 하나를 벗기면 의도하고 다르게 동작하나요..??👀

Reply:
괄호 하나 떼도 될 것 같네요 🫡